packages/@webex/plugin-cc/src/logger-proxy.ts

```
@@ -0,0 +1,48 @@
+      LoggerProxy.logger.error(LoggerProxy.format(LOGGING_LEVEL.error, message, context));
+    }
+  }
+
+  private static format(level: LOGGING_LEVEL, message: string, context: LogContext): string {
+    const timestamp = new Date().toISOString();
+    const moduleName = context.module || 'unknown';
+    const methodName = context.method || 'unknown';
+
+    return `${timestamp} ${LOG_PREFIX} - [${level}]: module:${moduleName} - method:${methodName} - ${message}`;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)